### PR TITLE
upt: add page

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 0.7.0

### CODE REVIEW

File: pages/common/upt.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# upt
+
+> Unified interface for managing packages across various operating systems, like Windows, many Linux distributions, Android, macOS, FreeBSD and even Haiku.
+> It requires the OS package manager to be installed.
```
Comment (fazlearefin) [inline pages/common/upt.md:4]:
```suggestion
> It requires the native OS package manager to be installed.
```

File: pages/common/upt.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# upt
+
+> Unified interface for managing packages across various operating systems, like Windows, many Linux distributions, Android, macOS, FreeBSD and even Haiku.
```
Comment (fazlearefin) [inline pages/common/upt.md:3]:
```suggestion
> Unified interface for managing packages across various operating systems, like Windows, many Linux distributions, macOS, FreeBSD and even Haiku.
```

Removed Android. The project mentions termux packages in Android, but termux is a third party tool and it does not get permissions to touch core android binaries. pkg packages are only meant for termux.

File: pages/common/upt.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+> It requires the OS package manager to be installed.
+> See also: `flatpak`, `brew`, `scoop`, `apt`.
```
Comment (fazlearefin) [inline pages/common/upt.md:5]:
```suggestion
> See also: `flatpak`, `brew`, `scoop`, `apt`, `dnf`.
```

`dnf` is quite popular for RedHat distros.

File: pages/common/upt.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- Upgrade all installed packages:
+
+`upt upgrade`
+
+- Upgrade a given package:
+
+`upt upgrade {{package}}`
+
+- List installed packages:
```
Comment (fazlearefin) [inline pages/common/upt.md:36]:
```suggestion
- List all installed packages:
```

Matching the wording from a previous example description.

Comment (vitorhcl) [reply pages/common/upt.md:36]: Well, "all" in the previous example description distinguishes `upt upgrade` from `upt upgrade {{package}}`. In this case, I don't think "all" is necessary.

**Review by fazlearefin — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**